shell/platform/android/io/flutter/view/FlutterMain.java

```diff
@@ -343,18 +345,69 @@ private static void initResources(Context applicationContext) {
         }
     }
 
+    /**
+     * Returns a list of the file names at the root of the application's 
+     * native library path.
+     */
+    private static Set<String> listLibs(Context applicationContext) {
+        ApplicationInfo applicationInfo = getApplicationInfo(applicationContext);
+        File[] files = new File(applicationInfo.nativeLibraryDir).listFiles();
+        
+        ImmutableSetBuilder builder = ImmutableSetBuilder.<String>newInstance();
+        for (File file : files) {
+            builder.add(file.getName());
+        }
+        return builder.build();
+    }
+
     private static void initAot(Context applicationContext) {
```

Comment:
It would be great if you could add comments to this method and any other that you think is relevant. Many developers won't have a clear notion of what is being loaded, where, and why. Any comments that we can provide that help illustrate the major moving pieces will be helpful for those developers. So in this case, what does it mean to init AOT? Does it always work the same way, or is it possible that this initialization process takes different paths, and why?

Reply:
Added a comment :)